concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+RonaTas-2011-431-1	1	carpenter	1042	CARPENTER
+RonaTas-2011-431-2	2	cannon, catapult	3202	CANNON
+RonaTas-2011-431-3	3	to fondle, pet, caress	3780	PET
+RonaTas-2011-431-4	4	stream, current	2935	CURRENT (STREAM)
+RonaTas-2011-431-5	5	false, imitation	3069	FALSE
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
+RonaTas-2011-431-9	9	to nurse, take care of	2472	SUCKLE (BREASTFEED, NURSE)
+RonaTas-2011-431-10	10	to decay, putrify, turn stale	2907	ROT
+RonaTas-2011-431-11	11	small, tiny	1246	SMALL
+RonaTas-2011-431-12	12	to mow	190	MOW
+RonaTas-2011-431-13	13	to decay, putrefy, go bad (of food, water)	2907	ROT
+RonaTas-2011-431-14	14	to harm	44	INJURE
+RonaTas-2011-431-15	15	a type of lasso		
+RonaTas-2011-431-16	16	ditch, canal	2002	DITCH
+RonaTas-2011-431-17	17	barley, Hordeum	932	BARLEY
+RonaTas-2011-431-18	18	barrow	2565	CASTRATED BOAR
+RonaTas-2011-431-19	19	gauntry, gantry (for supporting barrels)		
+RonaTas-2011-431-20	20	charm		
+RonaTas-2011-431-21	21	soggy place, moor, swamp, marsh	1145	SWAMP
+RonaTas-2011-431-22	22	hatchet, axe	677	AXE
+RonaTas-2011-431-23	23	pagan idol, stone or wooden pillar, vertical beam of different wooden instruments, strong, corpulent person	857	STONE
+RonaTas-2011-431-24	24	governor of Croatia, governor of the southern marches		
+RonaTas-2011-431-25	25	to regret, to be sorry	1875	REGRET
+RonaTas-2011-431-26	26	hairy, bushy (of dogs); a big hairy dog (as noun)		
+RonaTas-2011-431-27	27	label, tag		
+RonaTas-2011-431-28	28	cattle, livestock	1153	CATTLE
+RonaTas-2011-431-29	29	velvet		
+RonaTas-2011-431-30	30	to fuck		
+RonaTas-2011-431-31	31	courageous, brave, valiant	3	BRAVE
+RonaTas-2011-431-32	32	to estimate, to esteem, to appreciate		
+RonaTas-2011-431-33	33	frog	503	FROG
+RonaTas-2011-431-34	34	hobble, shackle, leg-iron		
+RonaTas-2011-431-35	35	stamp, bond, mark	1720	SIGN
+RonaTas-2011-431-36	36	wage, rent		
```

Comment:
This could be mapped to `1122 WAGES`.

Reply:
I'd leave it unmapped since WAGES is mapped to glosses like _Lohn_, _salary_, and not _rent_.

Reply:
Alright, will leave it unmapped then, just in case